conf/application.conf

```diff
@@ -128,7 +128,6 @@ urls {
   signOut = "http://localhost:9553/bas-gateway/sign-out-without-state"
   customsFinancialsHomepage = "http://localhost:9876/customs/payment-records"
   govUkHome = "https://www.gov.uk"
```

Comment:
@tyagikumarvineet , noticed few unused items in the config. Can you please remove these items?
govUkHome
application.cash-account.updateTime.intervalMilliseconds 
application.cash-account.updateTime.timeoutMilliseconds